source/js/components/StartButtons.js

```diff
@@ -18,19 +18,22 @@ class StartButtons extends HTMLElement {
   */
   constructor() {
     super();
-
     this.classList.add('row');
```

Comment:
If we change this to 'col' I think we can get rid of the button containers and have a few less divs